python/py-pastel/Portfile

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-pastel
+version             0.2.0
+revision            0
+categories-append   devel
+platforms           darwin
+license             MIT
+supported_archs     noarch
+
+python.versions     37 38
+
+maintainers         {gmail.com:davidgilman1 @dgilman} openmaintainer
+
+description         Bring colors to your terminal.
+
+long_description    Pastel is a simple library to help you colorize strings \
+                    in your terminal.
+
+homepage            https://github.com/sdispater/pastel
+
+checksums           rmd160  206c6be637d33defddc12bcf2ff76cef01980ae8 \
+                    sha256  46155fc523bdd4efcd450bbcb3f2b94a6e3b25edc0eb493e081104ad09e1ca36 \
+                    size    7601
+
+post-destroot {
+   # https://github.com/sdispater/pastel/issues/7
+   delete ${destroot}${python.pkgd}/tests
+}
```

Comment:
I think the issues that show up in Travis CI / Azure can be avoided if you move the ```post-destroot``` phase into the ```if-block``` below.